Source/town.cpp

```diff
@@ -148,9 +129,6 @@ void TownCloseGrave()
 	SetDungeonMicros();
 }
 
-/**
- * @brief Initialize all of the levels data
- */

```

Comment:
Do not delete documentation from local function that do not have a header entry.